bee-ternary/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.3.2-alpha - 2020-08-27
+
+### Added
+
+- `with_capacity` constructor for the buffers of every trit encoding.
+
```

Comment:
Can you make it a comment for now please ? I think we'll pack it with Josh's upcoming improvements

Reply:
sure